types/smb2/smb2-tests.ts

```diff
@@ -0,0 +1,4 @@
+function uhOh() {
+    // I don't have any tests!
+    // Quick, someone give me a samba server to connect to!
+}
```

Comment:
```suggestion
import SMB2 = require('smb2');

constsmb2Client = new SMB2({
  share:'\\\\000.000.000.000\\c$',
  domain:'DOMAIN',
  username:'username',
  password:'password!'
});
```
you can start with README, that `.ts` are not executed, they are compiled but do not emit.
you are not to connect to any server, please amend tests